package.json

```diff
@@ -92,7 +92,7 @@
     "prettier": "^2.2.1",
     "react-test-renderer": "^16.9.0",
     "relay-compiler": "^12.0.0",
-    "tab-e2e": "^1.1.0",
+    "tab-e2e": "^2.0.0-alpha.0",
```

Comment:
you'll need to modify **e2e-tests.test.js** ,which consumes this library and generates the actual tests ran at build time, to add retries.

change
```
describe('Tab: acceptance tests', () => {
  tests.forEach(({ description, test, testTimeout }) =>
    // eslint-disable-next-line jest/valid-title
    it(description, test, testTimeout)
  )
})
```

to
```
describe('Tab: acceptance tests', () => {
  jest.retryTimes(2)
  tests.forEach(({ description, test, testTimeout }) =>
    // eslint-disable-next-line jest/valid-title
    it(description, test, testTimeout)
  )
})
```